src/unix/aix/powerpc64.rs

```diff
@@ -256,6 +259,70 @@ s_no_extra_traits! {
         pub __pad: [c_int; 3],
     }
 
+    pub union _kernel_simple_lock {
+        pub _slock: c_long,
+        // Should be pointer to 'lock_data_instrumented'
+        pub _slockp: *mut c_void,
+    }
+
+    pub struct fileops_t {
+        pub fo_rw: extern "C" fn(
+            file: *mut file,
+            rw: crate::uio_rw,
+            io: *mut c_void,
+            ext: c_long,
+            secattr: *mut c_void,
+        ) -> c_int,
+        pub fo_ioctl: extern "C" fn(
+            file: *mut file,
+            a: c_long,
+            b: crate::caddr_t,
+            c: c_long,
+            d: c_long,
+        ) -> c_int,
+        pub fo_select:
+            extern "C" fn(file: *mut file, a: c_int, b: *mut c_ushort, c: extern "C" fn()) -> c_int,
+        pub fo_close: extern "C" fn(file: *mut file) -> c_int,
+        pub fo_fstat: extern "C" fn(file: *mut file, sstat: *mut crate::stat) -> c_int,
+    }
```

Comment:
Are the function pointers here ever null? They may need to be `Option<fn(...)>`

Also, are the headers/docs for any of these structs public?